### ISSUE TITLE
new

### ISSUE THREAD

**Issue by caiyinqiong** [caiyinqiong/Semantic-Retrieval-Models#1]
Learning Diverse Document Representations with Deep Query Interactions for Dense Retrieval（2022，多向量）
Topic-Grained Text Representation-based Model for Document Retrieval（2022，多向量）

**Comment by caiyinqiong** [comment]
pretraining for retrieval
- [ ] ConTextual Mask Auto-Encoder for Dense Passage Retrieval（预训练 for dense retrieval）
- [ ] Retromae: Pre-training retrieval-oriented transformers via masked auto-encoder（预训练 for dense retrieval）
- [ ] Simlm: Pre-training with representation bottleneck for dense passage retrieval.（预训练 for dense retrieval）
- [ ] LexMAE: Lexicon-Bottlenecked Pretraining for Large-Scale Retrieval（预训练 for dense retrieval）
- [ ] A Contrastive Pre-training Approach to Learn Discriminative Autoencoder for Dense Retrieval（预训练 for dense retrieval）

**Comment by caiyinqiong** [comment]
prompt  learning
- [ ] Scattered or Connected? An Optimized Parameter-efficient Tuning Approach for Information Retrieval
- [ ] DPTDR: Deep Prompt Tuning for Dense Passage Retrieval

**Comment by caiyinqiong** [comment]
LED: Lexicon-Enlightened Dense Retriever for Large-Scale Retrieval  （dense）
Aggretriever: A Simple Approach to Aggregate Textual Representation for Robust Dense Passage Retrieval（dense）

**Comment by caiyinqiong** [comment]
hybrid retrieval
UniﬁeR: A Uniﬁed Retriever for Large-Scale Retrieval

**Comment by caiyinqiong** [comment]
domain adaption：
Disentangled Modeling of Domain and Relevance for Adaptable Dense Retrieval

**Comment by caiyinqiong** [comment]
Isotropic Representation Can Improve Dense Retrieval（other）
Towards Robust Dense Retrieval via Local Ranking Alignment（other）

**Comment by caiyinqiong** [comment]
SpaDE: Improving Sparse Representations using a Dual Document Encoder for First-stage Retrieval（Expansion + Term Re-weighting）

**Comment by caiyinqiong** [comment]
zero-shot:
Promptagator: Few-shot Dense Retrieval From 8 Examples
